frontend/src/components/calendar_frontend/calendar-days.js

```diff
@@ -0,0 +1,43 @@
+function CalendarDays(props) {
+  const firstDayOfMonth = new Date(props.day.getFullYear(), props.day.getMonth(), 1);
+  const weekdayOfFirstDay = firstDayOfMonth.getDay();
+  let currentDays = [];
+
+  for (let day = 0; day < 42; day++) {
+    if (day === 0 && weekdayOfFirstDay === 0) {
+      firstDayOfMonth.setDate(firstDayOfMonth.getDate() - 7);
+    } else if (day === 0) {
+      firstDayOfMonth.setDate(firstDayOfMonth.getDate() + (day - weekdayOfFirstDay));
+    } else {
+      firstDayOfMonth.setDate(firstDayOfMonth.getDate() + 1);
+    }
+
+    let calendarDay = {
+      currentMonth: (firstDayOfMonth.getMonth() === props.day.getMonth()),
+      date: (new Date(firstDayOfMonth)),
+      month: firstDayOfMonth.getMonth(),
+      number: firstDayOfMonth.getDate(),
+      selected: (firstDayOfMonth.toDateString() === props.day.toDateString()),
+      year: firstDayOfMonth.getFullYear()
+    }
+
+    currentDays.push(calendarDay);
+  }
+
+  return (
```

Comment:
Since this function returns html element with javascript using `JSX`, it would help to rename this file to `calendar-days.jsx` to help other people know this file contains both HTML-like code and JavaScript logic